src/Examples/FontAwesome5.WinUI.Example/FontAwesome5.WinUI.Example.csproj

```diff
@@ -0,0 +1,41 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<OutputType>WinExe</OutputType>
+		<TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>
+		<TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
+		<RootNamespace>FontAwesome5.WinUI.Example</RootNamespace>
+		<ApplicationManifest>app.manifest</ApplicationManifest>
+		<Platforms>x86;x64;arm64</Platforms>
+		<RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
+		<PublishProfile>win10-$(Platform).pubxml</PublishProfile>
+		<UseWinUI>true</UseWinUI>
+		<EnablePreviewMsixTooling>true</EnablePreviewMsixTooling>
```

Comment:
i has to add `<WindowsPackageType>None</WindowsPackageType>` here otherwise i couldn't start the example app without a crash, can you confirm this?

Reply:
I don't get a crash. What version of VS are you using? I am using VS 2022 17.0.0

Reply:
Likely because the launchSettings.json was ignored by git but appears to be required to launch within Visual Studio.

![image](https://user-images.githubusercontent.com/22330242/142769650-c0900bd5-c471-47b6-a0f7-05859de57664.png)

Notice the `... (Package)` suffix is from the launch Settings profile.

It sounds like both options are possible. Either adding `WindowsPackageType` override or including the launchSettings.json file. Since the launchSettings.json is part of the WinUI 3 project template I would just keep the launchSettings.json file.

Reply:
I pushed commit to this PR that includes the launchSettings.json